src/components/wallet-select/index.tsx

```diff
@@ -30,30 +49,49 @@ export class WalletSelect extends React.Component<Properties> {
     if (this.props.isConnecting) {
       return (
         <div className='wallet-select__connecting-indicator'>
-          Connecting...
+          <Button label='Connecting...' />
         </div>
       );
     }
 
-    return this.renderWallets();
+    return (
+      <ul>
+        {this.renderWallets()}
+      </ul>
+    )
   }
 
   renderWallets() {
     return this.wallets.map(walletType => {
       const { type, name } = wallets[walletType];
+      const imageSource = [process.env.PUBLIC_URL, 'assets', providerConfig[type].imageSource].join('/');
```

Comment:
hmm. seems like this spans multiple repos? i think it'll be easier if we just add the images where they're used. additionally, env vars are substituted at build time, so i don't think use `PUBLIC_URL` here will do what we think it does for anyone else that is consuming this. eg. they're at the whim of whoever built this, and manages whatever that url is. (it might be working by accident due to the build chaining, but i don't think nested packages should depend on env vars in the consumer.)

Reply:
i think we should either let the consumer provide a configuration object that has/does not have images, or else just assume that we want these specific images here, and build them into this repo. someone can always submit a PR to make them conditional if they don't want them.